### ISSUE TITLE
Broken L2TP/IPSec after OpenVPN installed on Windows 11

### ISSUE THREAD

**Issue by Maxersh** [OpenVPN/openvpn#182]
**To Reproduce**
Install Windows 11 22H2 (on VirtualBox in my case).
Configure L2TP/IPSec VPN connection with pre-shared key in built-in Windows VPN client. All needed settings can be done via GUI.
Add Registry Key HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\PolicyAgent\AssumeUDPEncapsulationContextOnSendRule = 2
Reboot system.
Check that L2TP\IPSec connection can be established.
Install OpenVPN client community edition with default settings.
Check that L2TP\IPSec connection can not be established with error: "The L2TP connection attempt failed because the security layer encountered a processing error during the initial negotiations with the remote computer."
Uninstall OpenVPN client.
Check that L2TP\IPSec connection still can not be established with same error.

**Expected behavior**
L2TP/IPSec connection should work.

**Version information (please complete the following information):**
 - OS: Windows 11 22H2 (VirtualBox, official image)
 - OpenVPN version: 2.5.8

**Comment by schwabe** [comment]
OpenVPN does not support L2TP/IPSec. And as you wrote `Configure L2TP/IPSec VPN connection with pre-shared key in built-in Windows VPN client` you are using Windows internal VPN client. This does not

**Comment by cron2** [comment]
@lstipakov any idea how this could interfere?

**Comment by lstipakov** [comment]
No, we don't do anything at all related to Windows built-in VPN services. Probably one could get some more logs from Windows?

**Comment by cron2** [comment]
Since nothing ever happened in here for 3 years, I will now close the ticket.

Windows has progressed to 25H2, and it's known that Win11 releases broke and fixed many VPN problems over the time.  So it might just have fixed itself by means of Windows update.

That said, there is nothing in OpenVPN which would touch registry keys or VPN configs belonging to "other entities" so this is really nothing we could have fixed.
